vnext/Folly/TEMP_UntilFollyUpdate/concurrency/CacheLocality.cpp

```diff
@@ -0,0 +1,361 @@
+/*
```

Comment:
Could you document what is new here, or a link to an upstream PR?

Reply:
Removed this file, and replaced it with a fork of unistd.cpp, which is tracked with https://github.com/facebook/folly/issues/1452.

Reply:
Ended up with a fork here again:
https://github.com/facebook/folly/pull/1564 is the upstream PR.